### DISCUSSION TITLE
What tuning is suggested for the mediamtx:1.8.3-ffmpeg docker image or its host?

### DISCUSSION THREAD

**Discussion by moenodedev** [bluenviron/mediamtx#3526] [Questions]
### Question

We've deployed the `bluenviron/mediamtx:1.8.3-ffmpeg` to EKS on EC2. What settings should we configure in MediaMTX on the container or its host to support thousands of 4K streams?

**Comment by moenodedev** [comment]
We increased `WriteQueueSize` to `1024`.

**Reply by moenodedev** [reply]
We also increased the sysctl `net.core.rmem_max` setting on the host server of the container instance.

**Reply by aler9** [reply]
the `net.core.rmem_max` suggestion is interesting, maybe we can improve the server in order to set it automatically. More insights: https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes

**Reply by moenodedev** [reply]
Thanks I'll review their suggestions :)

**Comment by moenodedev** [comment]
Related
https://github.com/bluenviron/mediamtx/issues/3308#issuecomment-2195076934
